src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -777,26 +803,51 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
     if (runnableCalls.nonEmpty)
       logger.info(s"starting calls: " + runnableCalls.map(_.fullyQualifiedName).toSeq.sorted.mkString(", "))
 
-    val entries: Traversable[Try[ExecutionStartResult]] = runnableEntries map {
-      case (k: ScatterKey, _) => processRunnableScatter(k)
-      case (k: CollectorKey, _) => processRunnableCollector(k)
-      case (k: BackendCallKey, _) => processRunnableCall(k)
-      case (k: FinalCallKey, _) => processRunnableCall(k)
+    val (callKeyedRunnableEntries, nonCallKeyRunnableEntries) = runnableEntries.partition {
+      case (k: CallKey, _) => true
+      case _ => false
+    }
+
+    val callKeyEntries: Map[CallKey, Try[ExecutionStartResult]] = callKeyedRunnableEntries collect {
+      case (k: BackendCallKey, _) => k -> processRunnableCall(k)
+      case (k: FinalCallKey, _) => k -> processRunnableCall(k)
+      case (k: CallKey, v) =>
+        val message = s"Unknown entry in execution store:\nKEY: $k\nVALUE:$v"
```

Comment:
yeah that's probably how I noticed it :stuck_out_tongue:  don't feel obligated to fix it, but if you do great!